app/i18n/nl/feedback.php

```diff
@@ -120,7 +120,7 @@
 		'renamed' => 'Label „%s” hernoemd naar „%s”.',
 	),
 	'update' => array(
-		'can_apply' => 'FreshRSS word nu vernieud naar <strong>versie %s</strong>.',
+		'can_apply' => 'FreshRSS word nu vernieuwd naar <strong>versie %s</strong>.',
```

Comment:
Of bijgewerkt?

Reply:
Vind ik ook goed :) ik vind dat er wel meer 'ongebruikelijk' Nederlands wordt gebruikt in FreshRSS, daar zou misschien breder naar gekeken kunnen worden. Maar deze spelfout klopte sowieso niet.

Reply:
```suggestion
		'can_apply' => 'FreshRSS word nu bijgewerkt naar <strong>versie %s</strong>.',
```

Reply:
(The suggestion was so I could easily commit it from GH web UI.)